dijet/constants.py

```diff
@@ -9,16 +9,8 @@
     3.139,
     5.191,
 ]
-pt = [
-    70, 77, 87, 97,
-    111, 125, 145, 180,
-    210, 247, 280,
-    310, 335, 373,
-    410, 457,
-    500, 530, 562, 590,
-    650,
-    730, 790,
-    840,
-    920,
-    1020, 1120, 1240, 1400, 1700,
-]
+pt = [0, 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 35, 40,
+50, 55, 60, 65, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160,
+170, 180, 190, 200, 220, 240, 260, 280, 300, 325, 350, 375, 400,
+425, 450, 475, 500, 533, 566, 600, 633, 666, 700, 750, 800, 850,
+900, 1000, 3000]
```

Comment:
What is the reason for the change in the pT binning? I think we should stick to the default JER pT binning here for consistency reasons. If an extension to low/high pT or finer binning is needed for dedicated studies, we can add that via dedicated variables. @apaasch @yannick-fischer31 Could you check what the official pT binning the JER files are using and implement that here? Additional variables with a different binning can go in `config/variables.py`.